.github/workflows/release.yml

```diff
@@ -0,0 +1,112 @@
+# Builds the presto plugin and releases it to GitHub and Pypi
+name: Build and Release
+
+on:
+  workflow_dispatch:
+  
+# Release version number that must be updated for each release
+env:
+  version_number: '0.19.2'
```

Comment:
Is the "release" process just a commit that bumps this `version_number`?

Reply:
Do you mean this entire workflow? It does a couple things, it tests, bumps the version number, releases to the GitHub releases page, and uploads the release to pypi

Reply:
Oh sorry, I just meant—how do we trigger releases, using this workflow?